scripts/test_reports/report.py

```diff
@@ -419,11 +408,12 @@ def find_parent_in_junit(self, junit_xml, name):
     def get_case_text(self, text):
         return text
 
-
-
     def discover(self):
-
-        self.result_files.append({"file": self.results_file, "config": self.config})
+        for j in glob.glob("{}/*__zephyr*.xml".format(self.results_directory)):
```

Comment:
why are you keeping the same expectations on file names in the directory as with TCF? This could simply be <board>.xml and that is exactly what sanitycheck will be producing.

Reply:
We just decided with the team to use name like junit-name_of_board__commitdate_zephyr.xml
Sanitycheck is producing name test-report.xml which is absolutely the same for each board. But we store all results for each board in one folder. To avoid that, we decided to rename test-report.xml files when they are moving into one folder to be uploaded by modified script.